code/_compile_options.dm

```diff
@@ -85,9 +85,13 @@
 
 // If this is uncommented, will attempt to load and initialize prof.dll/libprof.so by default.
 // Even if it's not defined, you can pass "tracy" via -params in order to try to load it.
-// We do not ship byond-tracy. Build it yourself here: https://github.com/mafemergency/byond-tracy/
+// We do not ship byond-tracy. Build it yourself here: https://github.com/mafemergency/byond-tracy,
+// or the fork which writes profiling data to a file: https://github.com/ParadiseSS13/byond-tracy
 // #define USE_BYOND_TRACY
 
+// If uncommented, will display info about byond-tracy's status in the MC tab.
+// #define MC_TAB_TRACY_INFO
```

Comment:
for clarity I am asking you to lock anything outward facing behind USE_BYOND_TRACY, so the MC readout and the admin verb.
ideally the other parts of this pr would also be compile flags, since we are basically never going to enable byond tracy in repo

Reply:
It's worth noting that passing `-params tracy` to DreamDaemon (or in this pr, creating the `data/enable_tracy` file) will also enable byond-tracy, even without the `USE_BYOND_TRACY` define.